sci-chemistry/openbabel-perl/openbabel-perl-3.1.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit perl-module
```

Comment:
[I know better than to commit a Perl module without asking for @kentfredric  to check it over first!]